.gitignore

```diff
@@ -11,3 +11,6 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 .DS_Store
+
+# IDEs
+.idea
```

Comment:
nit, but that probably need to be set in your own gitignore rather than in the project one since this is coupled dev environments and not the project itself.

Reply:
💯 You exposed my laziness. 🥲 So now I must.

Reply:
Haha sorry, that's really a nit ^^